product_details/Fennec.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "Fennec",
+  "description": "Firefox for Android browser (deprecated)",
```

Comment:
If I read the tome of git changes over the last millennia correctly, this may actually be deprecated two times.